### PR TITLE
Make dnssimple compile on python 3

### PR DESCRIPTION
**Pull request by mscherer**
##### ISSUE TYPE
- Bugfix Pull Request
##### COMPONENT NAME

dnssimple

### CODE REVIEW

**Comment by gregdek** [comment]
Thanks @mscherer. To the current maintainers, @drcapulet please review according to guidelines (http://docs.ansible.com/ansible/developing_modules.html#module-checklist) and comment with text 'shipit', 'needs_revision' or 'close_me' as appropriate.

[This message brought to you by your friendly Ansibull-bot.]

File: network/dnsimple.py [hunk 1]
```diff
@@ -294,7 +294,8 @@ def main():
             else:
                 module.fail_json(msg="'%s' is an unknown value for the state argument" % state)
 
-    except DNSimpleException, e:
+    except DNSimpleException:
+        e = get_exception()
```
Comment (drcapulet) [inline network/dnsimple.py:298]: Do you need to import this?

Comment (mscherer) [reply network/dnsimple.py:298]: Yep, it work without (due to that code being already imported), but we should import something. I will add this.

**Review by drcapulet — APPROVED** [review]

**Comment by drcapulet** [comment]
LGTM, thanks! shipit

**Pull request merged**